internal/db/migrations/002_utility_functions.up.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION empty(t TEXT)
+    RETURNS BOOLEAN AS
+$empty$
+BEGIN
+    RETURN t ~ '^[[:space:]]*$';
+END;
+$empty$ LANGUAGE 'plpgsql';
+
+COMMIT;
```

Comment:
there's still quite a lot of end lines missing ^_^

Reply:
Fixed with `git ls-files -z | while IFS= read -rd '' f; do tail -c1 < "$f" | read -r _ || echo >> "$f"; done`

Added editorconfig for go project.